.github/tests/test_organization.py

```diff
@@ -123,6 +100,71 @@ def test_schema_naming(tmp_path):
             assert match.groups()
             version = match["version"]
 
+            assert "required" in loaded
+            # check "type" and "run" are required keys
+            assert "type" in loaded["required"]
+            assert "run" in loaded["required"]
+
+            # check all keys are properties
```

Comment:
Just to clarify - we are on the same page that these are required (or defined to be optional) currently for all schemas.

Reply:
oh that list was meant for script schema, but i see now that for loop is going over entire ``schema`` directory. Since there is only one schema this works, but it won't going forward with other schema files. I'd prefer to refactor this later because i would like to work with notion of writing a regtest per schema in a separate file like ``test_script.py``. Right now it was not clear which keys are global and which are not and i noticed when i started there were several key checks in ``test_organization.py`` so i figured i add all the checks in the same file. 

Maybe all the global keys shared by all schema should be tested in ``test_global.py`` and ``test_organization`` just checks the directory layout and names check (schema-v0.0.1.schema.json``).

I'd want to refactor these later in another PR.

Reply:
okay great - this sounds like a good plan, and what I wanted to talk about. We didn't have the concept of the global schema developed when I created test_organization.py, so I also agree about refactoring that file. But out of scope for this PR.